subprojects/core/src/integTest/groovy/org/gradle/api/ExternalScriptExecutionIntegrationSpec.groovy

```diff
@@ -17,15 +17,20 @@ package org.gradle.api
 
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 import org.gradle.test.fixtures.server.http.HttpServer
+import spock.lang.Unroll
 
 class ExternalScriptExecutionIntegrationSpec extends AbstractIntegrationSpec {
```

Comment:
@big-guy Do you mean credentials encoded in the URL? Do we currently support `apply from: 'https:...`?